src/geometry/shape_views/capsule.rs

```diff
@@ -76,9 +76,9 @@ impl_ref_methods!(CapsuleViewMut);
 
 impl<'a> CapsuleViewMut<'a> {
     /// Set the segment of this capsule.
-    pub fn set_segment(&mut self, a: Vect, b: Vect) {
-        self.raw.segment.a = a.into();
-        self.raw.segment.b = b.into();
+    pub fn set_segment(&mut self, axis: Vect, endpoint: Vect) {
```

Comment:
I feel like start/end would be less confusing here.

Reply:
I named renamed it to axis/endpoint because `parry3d::shape::Capsule` describes the segment field of a capsule as axis and endpoint. 
https://docs.rs/parry3d/0.13.1/parry3d/shape/struct.Capsule.html

Reply:
Hmm, fair enough, just seems a bit odd to me.

Reply:
I agree axis/endpoint are confusing here. The original documentation on `Capsule` doesn’t make sense, we’ll need to change it there (it should have been `The endpoints of the capsule’s principal axis.`).